code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -691,6 +818,46 @@
 		friends += fren
 		faction = fren.faction.Copy()
 
+/mob/living/simple_animal/hostile/proc/summon_backup(distance, exact_faction_match)
+	do_alert_animation()
+	playsound(loc, 'sound/machines/chime.ogg', 50, TRUE, -1)
+	for(var/mob/living/simple_animal/hostile/M in oview(distance, targets_from))
+		if(faction_check_mob(M, TRUE))
+			if(M.AIStatus == AI_OFF)
+				return
```

Comment:
```suggestion
				continue
```
Old code issue, might as well fix it while you're here (no clue why I didn't... maybe I did in Pink midnight+.)